docs/guide/cache/fastly.md

```diff
@@ -0,0 +1,467 @@
+
+!!! note "The Fastly Web Interface is not available for Ibexa Cloud"
+    It's recommend for Ibexa Cloud customers to use the Fastly CLI instead of using the Fastly API directly with `curl`, and so on.
+
+
+## Getting the Fastly credentials from an Ibexa Cloud installation
+
+In order to use the Fastly CLI or the Fastly API directly, you need to obtain the credentials for your site. The
+credentials can be obtained by ssh'ing into your Fastly-enabled environment (for example, production or staging) and
+type the command:
+
+```
+declare|grep FASTLY
+FASTLY_KEY=...
+FASTLY_SERVICE_ID=...
+```
+
+Note that these credentials will be different for your production and staging environments. When configuring the
+Fastly CLI, use the credentials for the environment you want to change.
+
+!!! note "Difference in environment variable names"
+    When configuring the Fastly CLI, you may use the environment variables `FASTLY_SERVICE_ID` and `FASTLY_API_TOKEN`, while
+    Ibexa DXP uses the variables `FASTLY_SERVICE_ID` and `FASTLY_KEY`.
+
+!!! note "Ibexa Cloud is delivered with Fastly already preconfigured"
+    Ibexa Cloud is delivered with Fastly preconfigured. That means that you don't have to do any changes to the Fastly
+    configuration in order to make your site work. The information provided here is only applicable if you want to change
+    the default Fastly configuration on Ibexa Cloud, or if you are not using Ibexa Cloud and want to configure Fastly
+    to work with Ibexa DXP on premise.
+
+!!! note "Varnish needs to be disabled when using Fastly"
+    Varnish is automatically provisioned on Ibexa Cloud. Varnish needs to be disabled on all environments which use
+    Fastly. See [documentation on how to do that](https://docs.platform.sh/guides/ibexa/fastly.html)
+
+## Quick setup of Fastly to use with Ibexa DXP
+
+Below are the few commands needed in order to install the required VCL configurtion for running Fastly with Ibexa DXP.
+In addition, you also need to set up domains, https and origin configuration (not covered here). What the various commands
+do are explained later on this page:
+
+```
+fastly vcl custom create --name "ez_main.vcl" --version=active --autoclone --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ --version=latest --main
+fastly vcl custom create --name "ez_user_hash.vcl" --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_hash.vcl --version=latest
+fastly vcl snippet create --name="Re-Enable shielding on restart" --version=latest --priority 100 --type recv --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/snippet_re_enable_shielding.vcl
+fastly service-version activate --version=latest
+```
+
+!!! note "None of the steps above are needed when using Ibexa Cloud as this is then preconfigured for you"
```

Comment:
```suggestion
!!! note "None of the steps are needed when you use Ibexa Cloud, because Fastly then comes preconfigured."
```

Also, please specify, which steps exactly - just the one command in this paragraph, or both "Get credentials..." and "Quickly set..."?
I would list the section titles explicitly to avoid confusion.